src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```diff
@@ -600,6 +600,10 @@ Navigation column (according to side column)
   display: unset;
 }
 
+.theme-switch-button span {
+  color: #fff;
```

Comment:
Wouldn't it be better to use the ``--pst-color-primary`` variable? So you "forget about it" in the next theme changes?

Reply:
I am using these variables for all items other than the navigation bar, which remains black (until we change its color). The primary color is different for both theme options, but we can define it to match the navigation color.

Reply:
Even with that, we should have a variable for basic colors. These could be later reused by the `--pst-color-primary` when declared for the light and dark themes.

For the moment, we can leave it as this. We are going to reformat the CSS very soon.